pkg/strategy/xmaker/strategy.go

```diff
@@ -1692,9 +1692,13 @@ func (s *Strategy) hedge(ctx context.Context, uncoveredPosition fixedpoint.Value
 
```

Comment:
Should add a return here?

Reply:
No, because spread maker will adjust the hedgeDelta